### PR TITLE
[Feature #147322191] Add social login on landing page, and registration page

### PR DESCRIPTION
**Pull request by daisymacharia**
#### What does this PR do?
Add social login buttons on landing page and registration page
#### Description of Task to be completed?
Allow the user access to social login button on several pages
#### How should this be manually tested?
The user can access the social login from the buttons provided
#### Screenshots (if appropriate)
screen shot 2017-07-11 at 10 07 23
screen shot 2017-07-11 at 10 07 52

### CODE REVIEW

**Comment by egichuri** [comment]
@daisymacharia for the homepage we might need to use lighter versions of the social login buttons since we have a dark background.

**Comment by daisymacharia** [comment]
image
How is this?

**Comment by egichuri** [comment]
looks good

**Review by egichuri — APPROVED** [review]

File: wger/core/views/user.py [hunk 1]
```diff
@@ -246,7 +246,7 @@ def registration(request):
     template_data['submit_text'] = _('Register')
     template_data['extend_template'] = 'base.html'
 
-    return render(request, 'form.html', template_data)
+    return render(request, 'reg_form.html', template_data)
```
Comment (egichuri) [inline wger/core/views/user.py:249]: This is fine but you could have inherited (extended) `form.html`

Comment (daisymacharia) [reply wger/core/views/user.py:249]: I had thought about that but there seemed to be other pages extending from form.html so I thought I could create a form specifically for the registration page

**Pull request merged**